composer.json

```diff
@@ -48,34 +48,34 @@
         "php": "^7.4 || ~8.0.0 || ~8.1.0",
         "ext-mongodb": "*",
         "container-interop/container-interop": "^1.2.0",
-        "doctrine/doctrine-laminas-hydrator": "^3.0.1",
+        "doctrine/doctrine-laminas-hydrator": "^3.1.0",
         "doctrine/doctrine-module": "^5.2.1",
-        "doctrine/event-manager": "^1.1.1",
-        "doctrine/mongodb-odm": "^2.3.2",
-        "laminas/laminas-eventmanager": "^3.4.0",
-        "laminas/laminas-modulemanager": "^2.11.0",
-        "laminas/laminas-mvc": "^3.3.3",
+        "doctrine/event-manager": "^1.2.0",
+        "doctrine/mongodb-odm": "^2.4.2",
+        "laminas/laminas-eventmanager": "^3.5.0",
+        "laminas/laminas-modulemanager": "^2.12.0",
+        "laminas/laminas-mvc": "^3.3.5",
         "laminas/laminas-servicemanager": "^3.10.0",
-        "laminas/laminas-stdlib": "^3.7.1",
-        "symfony/console": "^5.4.5 || ^6.0.5",
-        "symfony/var-dumper": "^5.4.6 || ^6.0.6"
+        "laminas/laminas-stdlib": "^3.13.0",
+        "symfony/console": "^5.4.15 || ^6.0.15",
+        "symfony/var-dumper": "^5.4.14 || ^6.0.14"
     },
     "require-dev": {
-        "doctrine/coding-standard": "^9.0.0",
-        "doctrine/persistence": "^2.4.0 || ^3.0.0",
+        "doctrine/coding-standard": "^9.0.2",
+        "doctrine/persistence": "^2.5.5 || ^3.0.4",
         "jangregor/phpstan-prophecy": "^1.0.0",
```

Comment:
Could this perhaps be dropped? :thinking:

Reply:
doctrine/persistence cannot be dropped, as it is used in:
- tests/Assets/CustomRepositoryFactory.php
- tests/Doctrine/ConfigurationFactoryTest.php
- as well as in the default module config

doctrine/coding-standard is needed for running CI with the adequate rules for PhpStand and Psalm if I am not mistaking?

Reply:
Ah, I see. This is because prophecy is used here:

https://github.com/doctrine/DoctrineMongoODMModule/blob/579f056f8306484f307ab2130f0c76878a263bb3/tests/Doctrine/DoctrineObjectHydratorFactoryTest.php#L32

Reply:
I know, but I didn't write that test case and I didn't want to refactor it either 🤷